### DISCUSSION TITLE
Help please...Maximum streaming connection speed only 1MB/s, screen freeze but sounds and controller works.

### DISCUSSION THREAD

**Discussion by froggyfly** [LizardByte/community#504] [Sunshine]
Hi,guys,
Can any friend help me point out how to troubleshoot my  streaming problem？

Moonlight+Sunshine streaming, LAN runs very well
A week ago, the external network connection was also running well and could run at 5MB/s or above.
Recently, it was found that the maximum connection speed is only 1MB/s. After opening the game or order a high Bitrate, the screen will freeze immediately, indicating a slow connection speed. At this time, the sound is playing normally, and the mouse and game controller are running normally and quickly.
Try using IPv6 and Zerotier, the problem is the same. reinstall, reset, anything I can do
Even I replaced the router with a brand new one, but it still remains the same.
Tested the network speed, Fully achieve the 1000Mbps download and 100mbps upload speeds claimed by the operator.

Try using Nvidia Shield instead of Sunshine, the problem is the same.
Today I found that using 'Parsec' can achieve the expected streaming speed, but Parsec streaming is not so good for me and having some login problem in my country.

Can any friends help me point out how to troubleshoot my problem,
Where could the error be caused?

**Comment by ReenigneArcher** [comment]
Do you have an issue when on LAN, or only when on WAN?

My suggestion is to lower the settings (resolution, bitrate, etc.)

**Reply by froggyfly** [reply]
only on WAN， but streaming is good before. 
Actually, even 720p resolution is not usable. 
I often use WAN to connect my LAN devices, so the network should be fast enough for straming Im sure.

**Reply by ReenigneArcher** [reply]
I think you are confusing what WAN and LAN are.

**Reply by froggyfly** [reply]
> I think you are confusing what WAN and LAN are.

Work good in LAN at home, but maximum connection speed is only 1MB/s on WAN outside.

Parsec streaming is well and has 5MB/s speed all the time.

[a comment by an automated account is not shown]
